pkg/doc/did/doc.go

```diff
@@ -428,3 +470,25 @@ func populateRawAuthentications(vms []VerificationMethod) []interface{} {
 
 	return rawAuthentications
 }
+
+func populateRawProofs(proofs []Proof) []interface{} {
+	var rawProofs []interface{}
+	var local Proof
+	for _, p := range proofs {
+		local = p
+		rawProofs = append(rawProofs, populateRawProof(&local))
+	}
+	return rawProofs
+}
+
+func populateRawProof(proof *Proof) map[string]interface{} {
```

Comment:
no need to use pointer `proof *Proof`  we have only read here

Reply:
Linter complained with error along the line: "struct is too big, pass by pointer" , that was the only reason I added it. I can revert the change and put nolint.

Reply:
Exact message: pkg/doc/did/doc.go:482:23: hugeParam: proof is heavy (88 bytes); consider passing it by pointer (gocritic)